R/mixture_model_wrapper.R

```diff
@@ -0,0 +1,141 @@
+#' Stack models according to CRPS
+#'
+#' @description Provides a wrapper for different EpiSoon models and generates
+#' a mixture model based on these models
+#'
+#' @param models A list of models. Models must be analogous to the form
+#' `function(...){EpiSoon::fable_model(model = , ...)}` or
+#' `function(...){EpiSoon::bsts_model(model = , ...)}`.
+#' @inheritParams bsts_model
+#' @return A dataframe of predictions (with columns representing the
+#'  time horizon and rows representing samples).
+#' @export
+#' @importFrom tsibble tsibble
+#' @importFrom fabletools model forecast
```

Comment:
This is missing most of the dplyr imports. A check across all deps would be useful

Reply:
do I add imports if I just use them in the example? e.g. the fable imports are not strictly necessary for the function.

Reply:
and I shouldn't add stackr to importFrom, right?

Reply:
nope just things used in the actual functions itself and yes dont import stackr